README.md

```diff
@@ -22,16 +22,16 @@ https://intelpython.github.io/dpnp/
 ## Dependencies
 
 * numba 0.52.* (IntelPython/numba)
-* dpCtl 0.6.*
-* dpNP >=0.5.1 (optional)
+* dpctl 0.6.*
+* dpnp >=0.5.1 (optional)
 * llvm-spirv (SPIRV generation from LLVM IR)
 * llvmdev (LLVM IR generation)
 * spirv-tools
 * scipy (for testing)
 
-## dpPy
+## dppy
 
-dpPy is a proof-of-concept backend for NUMBA to support compilation for
+dppy is a proof-of-concept backend for NUMBA to support compilation for
```

Comment:
```suggestion
dppy is a proof-of-concept backend for Numba to support compilation for
```

Reply:
We also should name Numba consistently. The name of product is `Numba`, but package name is `numba`.

Reply:
Done. I have fixed few more places in docs. More might remain.

Reply:
Please continue. Do not what is best with Numba. Possibly most cases should be `Numba`, but if we posint to pakcage name then `numba`.